### ISSUE TITLE
Deploy ETUmulatorWeb to Heroku

### ISSUE THREAD

**Issue by RootG** [kasirgalabs/ETUmulatorWeb#2]
In this issue you will deploy ETUmulatorWeb to [Heroku](https://www.heroku.com/) by using your account. Fork the project and setup an auto deployment to Heroku. After that open a pull request and include the link to your Heroku application. Easy, right?

However, there is no running services on ETUmulatorWeb. Therefore, we will not be able to show any web page when you deploy the project to Heroku. It would be nice if we had a **homepage** indicating that **ETUmulatorWeb is under development**. 

In short, building a service in order to show homepage is also a necessary part of this issue. You will need to figure out the implementation details of the service by yourself. The current version of source code able to run a server with default dependencies to implement this service.

There is a great tool called [Postman](https://www.getpostman.com/) for testing web services. It might help you while dealing with the issue.

**Comment by RootG** [comment]
#3

**Comment by RootG** [comment]
It is stated by Heroku help site GitHub repos that use submodules will generally not deploy correctly on Heroku. This is because GitHub does not include submodule contents when repo-content tarballs are generated.
